tests/actions/reposync_test.py

```diff
@@ -0,0 +1,251 @@
+import os
+import glob
+
+import pytest
+
+from cobbler import enums
+from cobbler.api import CobblerAPI
+from cobbler.cobbler_collections.manager import CollectionManager
+from cobbler.actions.reposync import RepoSync
+from cobbler.items.repo import Repo
+from cobbler import cexceptions
+from tests.conftest import does_not_raise
+
+
+@pytest.fixture(scope="class")
+def api():
+    return CobblerAPI()
+
+
+@pytest.fixture(scope="class")
+def reposync(api):
+    test_collection_mgr = CollectionManager(api)
+    test_reposync = RepoSync(test_collection_mgr, tries=2, nofail=False)
```

Comment:
I sadly did refactor this pattern a few commits ago. Thus you now need the API directly instead of the Collection Manager. I would recommend once more a rebase. Sorry for the trouble. If you don't want to do it, I will of course do it for you.

Reply:
@SchoolGuy Okay, I'll do a rebase.